src/core/auxiliar/session-token/getter.ts

```diff
@@ -0,0 +1,3 @@
+export function getSessionToken(): string {
+  return document.location.href.split('sessionToken=')[1];
```

Comment:
We can't assure that this will be the only parameter forever.
I rather suggest to use this code to obtain the `sessionToken` from Url.

```
const params = new URLSearchParams(window.location.search);
const sessionToken = params.get('sessionToken');
console.log(sessionToken)
```

Reply:
Nice suggestion, thank you!!